lib/fluent/plugin_helper/http_server.rb

```diff
@@ -34,11 +34,12 @@ module HttpServer
       # close    : correct stopped threads
       # terminate: kill thread
 
+      # @param title [Symbol] the thread name. this value should be unique.
       # @param addr [String] Listen address
       # @param port [String] Listen port
       # @param logger [Logger] logger used in this server
       # @param default_app [Object] This method must have #call.
-      def create_http_server(addr:, port:, logger:, default_app: nil)
+      def create_http_server(title:, addr:, port:, logger:, default_app: nil)
```

Comment:
Normal argument is better for `title` argument. `server` and other helpers use normal argument in 1st position.

Reply:
fixed https://github.com/fluent/fluentd/pull/2493/commits/2373dc30b30740b0469865c8fc93ad5cd542e748